### PR TITLE
Improve logging at SendMsgs and GetMsgResult

### PR DESCRIPTION
**Pull request by siburu**

### CODE REVIEW

File: pkg/relay/ethereum/tx.go [hunk 1]
```diff
@@ -48,49 +48,121 @@ func (c *Chain) SendMsgs(msgs []sdk.Msg) ([]core.MsgID, error) {
...
+		} else {
+			rawTxData = hex.EncodeToString(bz)
+		}
 		estimatedGas, err := c.client.EstimateGasFromTx(ctx, tx)
 		if err != nil {
-			revertReason, rawErrorData, err2 := c.getRevertReasonFromEstimateGas(err)
-			if err2 != nil {
-				logger.Error("failed to get revert reason", err2, "msg_index", i)
+			var revertReason, rawErrorData string
+			if reason, data, err := c.getRevertReasonFromEstimateGas(err); err != nil {
```
Comment (bluele) [inline pkg/relay/ethereum/tx.go:63]: @siburu We should include raw error data in the log even if the `ErrorRepository's parser function returns an error.

Comment (siburu) [reply pkg/relay/ethereum/tx.go:63]: I'm very sorry ... I fixed it.
https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/44/commits/f0ac44f10c48971e21a63b10ecfa2e02ea2f268a

File: pkg/relay/ethereum/tx.go [hunk 2]
```diff
@@ -48,49 +48,127 @@ func (c *Chain) SendMsgs(msgs []sdk.Msg) ([]core.MsgID, error) {
...
+				logAttrMaxGasLimit, c.Config().MaxGasLimit,
+				logAttrMsgIndex, i,
+				logAttrRawTxData, rawTxData,
+			)
 			txGasLimit = c.Config().MaxGasLimit
 		}
 		opts.GasLimit = txGasLimit
 		opts.NoSend = false
 		tx, err = c.SendTx(opts, msg, skipUpdateClientCommitment)
+		if bz, err := tx.MarshalBinary(); err != nil {
```
Comment (bluele) [inline pkg/relay/ethereum/tx.go:97]: @siburu  If the `tx` is nil(and err !=nil)  at line 96, doest it cause a nil reference error?

Comment (siburu) [reply pkg/relay/ethereum/tx.go:101]: @bluele I'm very sorry. I have fixed it. https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/44/commits/5178fa5a702cbd011e83c4f56bda378cd71db8c7

**Review by bluele — APPROVED** [review]
LGTM👍

**Review by mizuochik — APPROVED** [review]

**Pull request merged**